_includes/scenarios/integrate/request-and-response-introduction.md

```diff
@@ -0,0 +1,50 @@
+You want to share Attributes with a contact or receive Attributes from them?
+You need an authentication or a consent from a peer?
+You wish to make edits to a Relationship or ensure the correctness of data?
+All of these endeavors and more can be achieved by the means of Requests.
+They are the main instrument when it comes to interacting with other Identities going beyond the simple exchange of messages.
+
+Requests are always associated with a specific Relationship, i.e. they are sent to exactly one other Identity.
+Also, Responses are unique.
+Once a Request is answered, it cannot be answered again, e.g. on another device, if the App is installed on multiple ones, having all received the Request.
+This ensures an unambiguous functioning.
+
+## Types of Request- and ResponseItems
+
+The core of a [Request]({% link _docs_integrate/data-model-overview.md %}#request) is the [RequestItem]({% link _docs_integrate/data-model-overview.md %}#requestitem).
+It specifies what you want from the other Identity.
+There are many different types all described in detail in the [Requests and RequestItems]({% link _docs_integrate/requests-and-requestitems.md %}) section.
+Here, we just want to give a brief overview:
+
+- [AuthenticationRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#authenticationrequestitem)s can be used to request an authentication, e.g. for a login to a website
+- [ConsentRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#consentrequestitem)s can be used to request a consent to an freely configurable text, e.g. to receive a newsletter
+- [CreateAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#createattributerequestitem)s can be used to create an Identity- or RelationshipAttribute with a fixed value for a peer, e.g. a certificate
+- [FreeTextRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#freetextrequestitem)s can be used to send a free text to a contact, that in turn can accept this with a free text as well
+- [ProposeAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#proposeattributerequestitem)s can be used to ask the peer for Attributes, already suggesting answers, e.g. asking for an address and setting the country of the organization as default
+- [ReadAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#readattributerequestitem)s can be used to ask a peer for Attributes, e.g. asking for an address
+- [ShareAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#shareattributerequestitem)s can be used to share own IdentityAttributes with a contact, e.g. sharing the own DisplayName when requesting a new Relationship
```

Comment:
Just a small thought - we duplicate information here, and maybe we can also forget to update this list when new RequestItems are added or some are removed. Maybe the link to the page is enough.

Reply:
I thought about this as well. However, in my opinion the Requests and RequestItems Page is a bit overwhelming for someone new to the topic and I thought a brief overview in this introduction might be helpful. Of course, one must keep in mind to update this then, as well, if RequestItems change.